.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 exclude: '^(.*egg.info.*|.*/parameters.py|docs/).*$'
 default_language_version:
-    python: python3.8
+    python: python3.9
 repos:
```

Comment:
This would make the codebase unrunnable on Python 3.8 (which is still supported) because of some standard library import path changes that are invalid in 3.8.

Reply:
I see. nvm